example/images/models.py

```diff
@@ -33,4 +33,5 @@ def custom_rendition_property(self, info, **kwargs):
         GraphQLString("height", required=True),
         GraphQLImage("image", required=True),
         GraphQLString("file", required=True),
+        GraphQLString("lqip", required=True),
```

Comment:
note: I am not a fan of using `lqip` here. too much cognitive overhead. maybe `blur_hash` or `blur_placeholder`?

Reply:
blur_hash is great, but it might lead to confusion because of https://github.com/woltapp/blurhash 
Do you think this could b an issue ?

Reply:
heh, it could. `placeholder_blur` then?

Reply:
placeholder_blur it is then 👍